### ISSUE TITLE
Support for ~/ in output dir

### ISSUE THREAD

**Issue by phuhl** [gabm/Satty#63]
It seems that when using ~/ in the output dir path, no screenshot is saved and no error message is produced.

It would be good to be able to use the ~ shortcut for the home directory of the user.

**Comment by gabm** [comment]
yeah, there is currently no path expansion.. sure there is no message? if there is a positive message, maybe you have a file on your hdd somewhere that starts with ~/?

**Comment by gabm** [comment]
okay.. nevertheless: path expansion might be nice

**Comment by gabm** [comment]
here are a number of alternatives: https://stackoverflow.com/questions/54267608/expand-tilde-in-rust-path-idiomatically

**Comment by xlucn** [comment]
Shell expand is a good option. That allows even more customization, like `$HOME`.
